fog-core.gemspec

```diff
@@ -21,7 +21,9 @@ Gem::Specification.new do |spec|
   spec.add_dependency("builder")
   spec.add_dependency("excon", "~> 0.49")
   spec.add_dependency("formatador", "~> 0.2")
+  spec.add_dependency("xmlrpc") if RUBY_VERSION.to_s >= "2.4"
 
+  spec.add_development_dependency("tins") if RUBY_VERSION.to_s > "2.0"
```

Comment:
I'm actually not sure but under the Gemfile.lock it's under `coveralls` so I figure it was safe to add explicitly as a development dependency. This is to fix a travis error on rubies older than 2. I don't mind rm/moving this around, whatever makes Travis happy.